apps/docs/docs/user/composite-blocks.md

```diff
@@ -0,0 +1,115 @@
+---
+sidebar_position: 4
+---
+
+# Composite Blocks
+
+Composite blocks are a way to create new blocktypes in Jayvee by combining the functionality of existing blocks and pipes. By relying on composite blocks instead of implementing more builtin blocks in a language interpreter Jayvee supports easy extension by users.
```

Comment:
```suggestion
Composite blocks are a way to create new blocktypes in Jayvee by combining the functionality of existing blocks and pipes. By relying on composite blocks instead of implementing more builtin blocks in a language interpreter, Jayvee supports easy extension by users.
```